source/tyr/tyr/binarisation.py

```diff
@@ -697,8 +697,10 @@ def get_bano2mimir_params(working_directory, autocomplete_instance, autocomplete
         working_directory,
         '-m',
         current_app.config['MIMIR_PLATFORM_TAG'],
-        'run',
     ]
+    if cosmogony_file is not None:
+        params.extend(['-s', 'cosmogony_file=\'{}\''.format(cosmogony_file)])
+    return params.extend(['run'])
```

Comment:
I'm not sure this is going to work because `extend()` function doesn't return anything, right? You probably need to do it in two lines.
```suggestion
    params.extend(['run'])
    return params
```

The same should be fixed below.

Reply:
You are absolutely right. thanks